polygeist/lib/Dialect/Polygeist/Transforms/LICM.cpp

```diff
@@ -352,26 +353,21 @@ class LICMCandidate {
 };
 
 //===----------------------------------------------------------------------===//
-// SYCLAccessorVersionBuilder
+// VersionConditionBuilder
 //===----------------------------------------------------------------------===//
 
-// TODO: Support other kind of loops.
-class SYCLAccessorVersionBuilder : public SCFLoopVersionBuilder {
+class VersionConditionBuilder {
```

Comment:
As the version builder takes accessor pairs, so it is particularly for SYCLAccessor, do you think is more suitable to have that in the class name?

Reply:
I think is already clear from the constructor signature.

Reply:
ok